docker/nginx/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openresty/openresty:1.19.3.1-2-bionic
+
+# RUN apt-get update -qq && apt-get install cron logrotate -qq
```

Comment:
are you purposefully committing this?

Reply:
from offline: "it's for future reference - that's to support the todos I described in pr description"